src/api/wayfire/signal-definitions.hpp

```diff
@@ -569,6 +569,16 @@ struct view_resize_request_signal : public _view_signal
     uint32_t edges;
 };
 
+/**
+ * view-hints-changed signal is emitted on the view's output whenever the client
+ * indicates the views hints have changed (example urgency hint).
+ */
+struct wf::view_hints_changed_signal : public _view_signal
```

Comment:
```suggestion
struct view_hints_changed_signal : public _view_signal
```
The `wf::` is unnecessary because the whole file is inside `namespace wf{}`